Submitted By: Steve Jones sljones3@arkansas.net
Date: 2003-10-02
Initial Package Version: 1.5k
Origin: Steve Jones sljones3@arkansas.net
	http://archives.linuxfromscratch.org/mail-archives/lfs-dev/2002-October/029607.html
Description: Fixes man page width formating issues on terminal widths other 
	than 80 collumns with versions of groff greater than groff-1.18
diff -Naur man-1.5l.orig/src/man.c man-1.5l/src/man.c
--- man-1.5l.orig/src/man.c	2002-12-17 16:28:14.000000000 -0700
+++ man-1.5l/src/man.c	2003-04-24 16:49:05.000000000 -0600
@@ -546,7 +546,9 @@
 	       /* we should set line length and title line length */
 	       /* however, a .lt command here fails, only
 		  .ev 1; .lt ...; .ev helps for my version of groff */
-	       sprintf(eos(bufh), "echo \".ll %d.%di\"; ", ll/10, ll%10);
+		  /* use the LL register instead of .ll to keep up with
+		  * changes in the groff-1.18 series */
+		  sprintf(eos(bufh), "echo \".nr LL %d.%di\"; ", ll/10, ll%10);
 #if 0
 	       sprintf(eos(bufh), "echo \".lt %d.%di\"; ", ll/10, ll%10);
 #endif
